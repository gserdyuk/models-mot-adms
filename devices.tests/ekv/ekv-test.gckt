* test circuit ekv
*.options drop_spice_comments dollar_as_spice_comment

.include ./ekv26_0u5.noc.par
*MCKT 1 2 3 4 nekv26eval W = 2e-05 L = 2e-05
MCKT 1 2 3 4 nmos W = 2e-05 L = 2e-05

* DEF SOURCES
VG 2 0 DC 0
VS 3 0 DC 0
VD 1 0 DC 0.05
VB 4 0 DC 0

* END SOURCES

.PRINT DC v(VG)  v(VB)  I(VD) 
.DC VG 0 2.51 0.1 VB 0 -1.05 -0.5 basic


.end
