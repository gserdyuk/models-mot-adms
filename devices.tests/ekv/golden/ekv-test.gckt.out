Gnucap 2009.12.07 RCS 26.136
The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2009, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
M: already installed, replacing
stashing as M:0
* test circuit ekv 
#           v(VG)      v(VB)      I(VD)     
 0.         0.         0.         0.        
 0.1        0.1        0.         0.        
 0.2        0.2        0.         0.        
 0.3        0.3        0.         0.        
 0.4        0.4        0.        -755.4E-12 
 0.5        0.5        0.        -11.495E-9 
 0.6        0.6        0.        -117.68E-9 
 0.7        0.7        0.        -467.8E-9  
 0.8        0.8        0.        -978.13E-9 
 0.9        0.9        0.        -1.5432E-6 
 1.         1.         0.        -2.1231E-6 
 1.1        1.1        0.        -2.7027E-6 
 1.2        1.2        0.        -3.2751E-6 
 1.3        1.3        0.        -3.837E-6  
 1.4        1.4        0.        -4.3869E-6 
 1.5        1.5        0.        -4.924E-6  
 1.6        1.6        0.        -5.448E-6  
 1.7        1.7        0.        -5.959E-6  
 1.8        1.8        0.        -6.457E-6  
 1.9        1.9        0.        -6.9425E-6 
 2.         2.         0.        -7.4156E-6 
 2.1        2.1        0.        -7.8768E-6 
 2.2        2.2        0.        -8.3263E-6 
 2.3        2.3        0.        -8.7647E-6 
 2.4        2.4        0.        -9.1922E-6 
 2.5        2.5        0.        -9.6092E-6 
 0.         0.        -0.5        0.        
 0.1        0.1       -0.5        0.        
 0.2        0.2       -0.5        0.        
 0.3        0.3       -0.5        0.        
 0.4        0.4       -0.5        0.        
 0.5        0.5       -0.5        0.        
 0.6        0.6       -0.5       -1.5097E-9 
 0.7        0.7       -0.5       -25.53E-9  
 0.8        0.8       -0.5       -209.E-9   
 0.9        0.9       -0.5       -629.68E-9 
 1.         1.        -0.5       -1.1561E-6 
 1.1        1.1       -0.5       -1.7153E-6 
 1.2        1.2       -0.5       -2.2817E-6 
 1.3        1.3       -0.5       -2.845E-6  
 1.4        1.4       -0.5       -3.4004E-6 
 1.5        1.5       -0.5       -3.9456E-6 
 1.6        1.6       -0.5       -4.4794E-6 
 1.7        1.7       -0.5       -5.0012E-6 
 1.8        1.8       -0.5       -5.5108E-6 
 1.9        1.9       -0.5       -6.0082E-6 
 2.         2.        -0.5       -6.4936E-6 
 2.1        2.1       -0.5       -6.9672E-6 
 2.2        2.2       -0.5       -7.4293E-6 
 2.3        2.3       -0.5       -7.8802E-6 
 2.4        2.4       -0.5       -8.3201E-6 
 2.5        2.5       -0.5       -8.7495E-6 
 0.         0.        -1.         0.        
 0.1        0.1       -1.         0.        
 0.2        0.2       -1.         0.        
 0.3        0.3       -1.         0.        
 0.4        0.4       -1.         0.        
 0.5        0.5       -1.         0.        
 0.6        0.6       -1.         0.        
 0.7        0.7       -1.         0.        
 0.8        0.8       -1.        -8.1883E-9 
 0.9        0.9       -1.        -105.27E-9 
 1.         1.        -1.        -445.26E-9 
 1.1        1.1       -1.        -934.93E-9 
 1.2        1.2       -1.        -1.473E-6  
 1.3        1.3       -1.        -2.0244E-6 
 1.4        1.4       -1.        -2.5759E-6 
 1.5        1.5       -1.        -3.1215E-6 
 1.6        1.6       -1.        -3.6583E-6 
 1.7        1.7       -1.        -4.1849E-6 
 1.8        1.8       -1.        -4.7005E-6 
 1.9        1.9       -1.        -5.2047E-6 
 2.         2.        -1.        -5.6975E-6 
 2.1        2.1       -1.        -6.1789E-6 
 2.2        2.2       -1.        -6.6491E-6 
 2.3        2.3       -1.        -7.1083E-6 
 2.4        2.4       -1.        -7.5568E-6 
 2.5        2.5       -1.        -7.9948E-6 
